NativeUI/PauseMenu/TabItemSimpleList.cs

```diff
@@ -29,13 +30,13 @@ public override void Draw()
 
             for (int i = 0; i < Dictionary.Count; i++)
             {
-                new UIResRectangle(new Point(TopLeft.X, TopLeft.Y + (40 * i)),
+                new NativeRectangle(new Point(TopLeft.X, TopLeft.Y + (40 * i)),
                     new Size(rectSize, 40), i % 2 == 0 ? Color.FromArgb(alpha, 0, 0, 0) : Color.FromArgb(blackAlpha, 0, 0, 0)).Draw();
 
                 var item = Dictionary.ElementAt(i);
 
-                new UIResText(item.Key, new Point(TopLeft.X + 6, TopLeft.Y + 5 + (40 * i)), 0.35f, Color.FromArgb(fullAlpha, Color.White)).Draw();
-                new UIResText(item.Value, new Point(BottomRight.X - 6, TopLeft.Y + 5 + (40 * i)), 0.35f, Color.FromArgb(fullAlpha, Color.White), Font.ChaletLondon, UIResText.Alignment.Right).Draw();
+                new NativeText(item.Key, new Point(TopLeft.X + 6, TopLeft.Y + 5 + (40 * i)), 0.35f, Color.FromArgb(fullAlpha, Color.White)).Draw();
+                new NativeText(item.Value, new Point(BottomRight.X - 6, TopLeft.Y + 5 + (40 * i)), 0.35f, Color.FromArgb(fullAlpha, Color.White), Font.ChaletLondon, TextAlignment.Right).Draw();
```

Comment:
For future optimizations this should be moved into the constructor like it's done in UIMenu and alike.
Also: maybe events for stuff like window/safezone resize so elements can change accordingly without checking every frame?

Reply:
I know that there is a lot of stuff that needs organization and optimization, but as the PR says is just the important classes that end up doing the drawing work.